Package.swift

```diff
@@ -17,7 +17,8 @@ import PackageDescription
 import class Foundation.ProcessInfo
 
 let upcomingFeatureSwiftSettings: [SwiftSetting] = [
-    .enableUpcomingFeature("ExistentialAny")
+    .enableUpcomingFeature("ExistentialAny"),
+    .enableUpcomingFeature("StrictConcurrency"),
```

Comment:
We probably don't want this unilaterally: either only in CI, or only in the Swift 6 compiler.

Reply:
Oh nevermind, for ASN.1 it's totally fine as it doesn't have any concurrency anyway.